library/std/src/ffi/c_str.rs

```diff
@@ -613,7 +613,8 @@ impl CString {
     #[inline]
     #[stable(feature = "rust1", since = "1.0.0")]
     pub fn as_bytes(&self) -> &[u8] {
-        &self.inner[..self.inner.len() - 1]
+        // SAFETY: CString has a length at least 1
+        unsafe { &self.inner.get_unchecked(..self.inner.len() - 1) }
```

Comment:
`get_unchecked` already returns a reference, so no need for the `&`:
```suggestion
        unsafe { self.inner.get_unchecked(..self.inner.len() - 1) }
```